distmv/normal.go

```diff
@@ -118,3 +122,107 @@ func (n *Normal) Rand(x []float64) []float64 {
 	floats.Add(x, n.mu)
 	return x
 }
+
+// ConditionNormal returns the Normal distribution that is the receiver conditioned
+// on the input evidence. The returned multivariate normal has dimension
+// n - len(observed), where n is the dimension of the original receiver. The updated
+// mean and covariance are
+//  mu = mu_1 + sigma_{2,1}^T * sigma_{2,2}^-1 (v - mu_2)
+//  sigma = sigma_{1,1} - sigma_{2,1}^T * sigma_{2,2}^-1 * sigma_{2,1}
+// where the elements of x_2 have been observed with values v. The dimension order
```

Comment:
Maybe the notation is very standard in statistics but the comment is confusing for me without looking at Wikipedia. I like the comments to be as self-contained as reasonable which in this case is difficult but could you please at least write something about the subscripts and how they relate to observed?